### PR TITLE
Metadata window width

### LINKED ISSUE #109: Edit metadata window is huge when path is long

### PR DESCRIPTION
**Pull request by azfoo**
Closes #109.

### CODE REVIEW

File: tilia/settings.py [hunk 1]
```diff
@@ -12,6 +12,9 @@
         "window_y": 10,
         "timeline_bg": "#EEE",
     },
+    "metadata": {
+        "window-width": 400
+    },
```
Comment (FelipeDefensor) [inline tilia/settings.py:17]: We already have a "media_metadata" section, right below. Could we use that?

File: tilia/ui/windows/metadata.py [hunk 2]
```diff
@@ -32,6 +33,7 @@ class MediaMetadataWindow(QDialog):
     def __init__(self):
         super().__init__()
         self.setWindowTitle("Metadata")
+        self.setFixedWidth(settings.get("metadata", "window-width"))
```
Comment (FelipeDefensor) [inline tilia/ui/windows/metadata.py:36]: This is good, but if there's no reason not to we should go with `self.resize(settings.get("metadata", "window-width"), self.height())` to keep the window resizable.

**Review by FelipeDefensor — CHANGES_REQUESTED** [review]

File: tilia/settings.py (hunk 1 above)
Comment (azfoo) [reply tilia/settings.py:17]: Good call

File: tilia/ui/windows/metadata.py (hunk 2 above)
Comment (azfoo) [reply tilia/ui/windows/metadata.py:36]: Fixed Width:
image

Regular Resize:
image

Word wrapping is not very clever - someone with a silly file name could still accidentally make the window very long. But if that's compromisable, I agree that regular resizing provides more flexibility.

Comment (FelipeDefensor) [reply tilia/ui/windows/metadata.py:36]: Oh, I see. I still slightly prefer the resize option. In the future, we could add some processing to the string to handle those silly cases.

**Review by FelipeDefensor — APPROVED** [review]

**Pull request merged**